src/modules/coreHttpApi/controllers/AttributesController.ts

```diff
@@ -214,6 +214,34 @@ export class AttributesController extends BaseController {
         return this.ok(result);
     }
 
+    @DELETE
+    @Path("/Own/Shared/:id")
+    public async deleteOwnSharedAttributeAndNotifyPeer(@PathParam("id") attributeId: string): Promise<Envelope> {
+        const result = await this.consumptionServices.attributes.deleteOwnSharedAttributeAndNotifyPeer({ attributeId });
+        return this.ok(result);
+    }
+
+    @DELETE
+    @Path("/Peer/Shared/:id")
+    public async deletePeerSharedAttributeAndNotifyOwner(@PathParam("id") attributeId: string): Promise<Envelope> {
+        const result = await this.consumptionServices.attributes.deletePeerSharedAttributeAndNotifyOwner({ attributeId });
+        return this.ok(result);
+    }
+
+    @DELETE
+    @Path("/Third/Party/:id")
```

Comment:
I am a bit confused, why not `/ThirdParty/:id`? The words third and party do not make any sense when not combined in this context